src/notation/qml/MuseScore/NotationScene/ViewModeControl.qml

```diff
@@ -62,6 +62,8 @@ Item {
                 return
             }
 
+            menu.model = 0
```

Comment:
better to use the clear function for model

Reply:
Did not understand the suggestion. Are you suggesting adding a function?
```
function clear() {
    menu.model = 0
}
```

Reply:
No, I mean menu.model.clear()

Reply:
No, I mean menu.model.clear()